db/upgrade.php

```diff
@@ -130,5 +130,39 @@ function xmldb_accredible_upgrade($oldversion=0) {
 
     }
 
+    if ($oldversion < 2022053000) {
+        $table = new xmldb_table('accredible');
+
+        // Define field includegradeattribute to be added to accredible.
+        $field = new xmldb_field('includegradeattribute', XMLDB_TYPE_INTEGER, '1', null, null, null, '0',
```

Comment:
Should we add `XMLDB_NOTNULL` to `includegradeattribute` since it's a boolean with a default value?